figure_notebooks/figure1.md

```diff
@@ -269,7 +269,7 @@ full_image <- image_append(image_scale(c(bottom_panel, news_nature_gg),3000),
 print(full_image)
```

Comment:
According to your comments you removed the guardian from the analysis; however, the guardian category here still shows up in the figure. Is this intended?

Reply:
oops! no, legend is fixed now.

Reply:
https://github.com/greenelab/nature_news_disparities/pull/34/commits/aa55aa31cce937cf99080c9b06caa574fa178d61